### PR TITLE
Remove collectd setup via bundle install and omsadmin.sh

### PR DESCRIPTION
**Pull request by sugr4**
Modify docs for collectd manual setup

@Microsoft/omsagent-devs @agup006

### CODE REVIEW

File: docs/OMS-Agent-for-Linux.md [hunk 1]
```diff
@@ -547,16 +546,32 @@ The OMS Agent for Linux also listens on port 26000 for CollectD metrics and then
...
+    ```
+    If your CollectD config directory is located in `/etc/collectd/collectd.conf.d/`:
+    ```
+    sudo cp /etc/opt/microsoft/omsagent/sysconf/omsagent.d/oms.conf /etc/collectd/collectd.conf.d/oms.conf
+    ```
+    **Note:** For CollectD versions before 5.5 you will have to modify the tags in `oms.conf` as shown above.
+
+2. Copy `collectd.conf` to the desired workspace's omsagent configuration directory.
+    ```
+    sudo cp /etc/opt/microsoft/omsagent/sysconf/omsagent.d/collectd.conf /etc/opt/microsoft/omsagent/<workspace id>/conf/omsagent.d/
```
Comment (lagalbra) [inline docs/OMS-Agent-for-Linux.md]: Include the command "sudo chown omsagent:omiusers /etc/opt/microsoft/omsagent/<workspace id>/conf/omsagent.d/collectd.conf"

File: installer/bundle/bundle_skel.sh [hunk 2]
```diff
@@ -553,16 +552,6 @@ do
             shift 1
             ;;
 
-        --collectd)
```
Comment (lagalbra) [inline installer/bundle/bundle_skel.sh]: Maybe we could keep this condition, but if this parameter is used, just print an informational message like "CollectD can no longer be configured with the --collectd option. Please see OMSAgent-for-Linux documentation here: https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/docs/OMS-Agent-for-Linux.md"

File: installer/scripts/omsadmin.sh [hunk 3]
```diff
@@ -236,10 +232,6 @@ parse_args()
         a)
             AZURE_RESOURCE_ID=$OPTARG
             ;;
-        c) 
-            COLLECTD=1
```
Comment (lagalbra) [inline installer/scripts/omsadmin.sh]: Same printout idea here: print an informational message like "CollectD can no longer be configured with the --collectd option. Please see OMSAgent-for-Linux documentation here: https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/docs/OMS-Agent-for-Linux.md"

File: docs/OMS-Agent-for-Linux.md (hunk 1 above)
Comment (sugr4) [inline docs/OMS-Agent-for-Linux.md]: Done

File: installer/bundle/bundle_skel.sh (hunk 2 above)
Comment (sugr4) [inline installer/bundle/bundle_skel.sh]: I don't think this is necessary as I expect this message to be a part of the release notes.

**Pull request merged**